common/djangoapps/student/views.py

```diff
@@ -1740,6 +1747,10 @@ def create_account_with_params(request, params):
     if third_party_auth.is_enabled() and pipeline.running(request):
         running_pipeline = pipeline.get(request)
         third_party_provider = provider.Registry.get_from_pipeline(running_pipeline)
+        # EnterpriseCustomers (part of the `enterprise` app) tied to a third-party
```

Comment:
I'm not really understanding this comment -- is there an actual configuration parameter/switch that determines whether or not "consent permissions are saved to the pipeline for storage elsewhere"?